public/pages/Destinations/containers/CreateDestination/CreateDestination.js

```diff
@@ -170,42 +170,44 @@ class CreateDestination extends React.Component {
               <EuiSpacer size="m" />
               <ContentPanel title="Destination" titleSize="s" bodyStyles={{ padding: 'initial' }}>
                 <div style={{ padding: '0px 10px' }}>
-                  <FormikFieldText
-                    name="name"
-                    formRow
-                    fieldProps={{
-                      validate: validateDestinationName(
-                        httpClient,
-                        _.get(location, 'state.destinationToEdit')
-                      ),
-                    }}
-                    rowProps={{
-                      label: 'Name',
-                      helpText: 'Specify a name of the destination.',
-                      style: { paddingLeft: '10px' },
-                      isInvalid,
-                      error: hasError,
-                    }}
-                    inputProps={{
-                      isInvalid,
-                    }}
-                  />
-                  <FormikSelect
-                    name="type"
-                    formRow
-                    rowProps={{
-                      label: 'Type',
-                      style: { paddingLeft: '10px' },
-                    }}
-                    inputProps={{
-                      disabled: edit,
-                      options: DESTINATION_OPTIONS,
-                    }}
-                  />
+                  <div style={{ padding: '0px 10px' }}>
+                    <FormikFieldText
+                      name="name"
+                      formRow
+                      fieldProps={{
+                        validate: validateDestinationName(
+                          httpClient,
+                          _.get(location, 'state.destinationToEdit')
+                        ),
+                      }}
+                      rowProps={{
+                        label: 'Name',
+                        helpText: 'Specify a name of the destination.',
+                        isInvalid,
+                        error: hasError,
+                      }}
+                      inputProps={{
+                        isInvalid,
+                      }}
+                    />
+                    <FormikSelect
+                      name="type"
+                      formRow
+                      rowProps={{
+                        label: 'Type',
+                      }}
+                      inputProps={{
+                        disabled: edit,
+                        options: DESTINATION_OPTIONS,
+                      }}
+                    />
+                  </div>
```

Comment:
For the above, I added a `<div style={{ padding: '0px 10px' }}>` instead of assign the CSS style for the 2 elements separately.